runatlantis.io/docs/server-configuration.md

```diff
@@ -305,6 +305,15 @@ and set `--autoplan-modules` to `false`.
   This means that an attacker could spoof calls to Atlantis and cause it to perform malicious actions.
   :::
 
+### `--checkout-depth`
+  ```bash
+  atlantis server --checkout-depth=0
+  # or
+  ATLANTIS_CHECKOUT_DEPTH=0
+  ```
+  The number of commits to fetch from the branch. Used if `--checkout-strategy=merge` since the `--,checkout-strategy=branch` (default) checkout strategy always defaults to a shallow clone using a depth of 1.
```

Comment:
```suggestion
  The number of commits to fetch from the branch. Used if `--checkout-strategy=merge` since the `--checkout-strategy=branch` (default) checkout strategy always defaults to a shallow clone using a depth of 1.
```